download-pmd.sh

```diff
@@ -5,13 +5,70 @@
 # TODO: 1. Get PMD_VERSION from package.json
 # TODO: 2. What happens when pmd binary cannot be downloaded?
 
+
+usage(){
+	echo "
+	usage: $0 [options]
+	$0 downloads and sets up PMD
+	OPTIONS:
+    -h | --help         Shows the usage information
+    -f | --force        Forces download of PMD. Default behavior is to download/setup PMD only if needed  
+	"
+}
+
+FORCE=false
+REQUIRES_SETUP=true
+
+# Parse arguments
+while [ "$1" != "" ]
+do
+	case $1 in
+		-h | --help ) usage
+		exit 0
+		;;
+
+		-f | --force ) shift
+		FORCE=true
+		;;
+
+		*	) usage
+		exit 2
+		;;
+
+	esac
+	shift
+done
+
+
+
 PMD_VERSION=6.20.0
 PMD_FILE=pmd-bin-${PMD_VERSION}
 PMD_URL=https://github.com/pmd/pmd/releases/download/pmd_releases%2F${PMD_VERSION}/${PMD_FILE}.zip
 DIST_DIR=`pwd`/dist
 PMD_ZIP=${DIST_DIR}/${PMD_FILE}.zip
 PMD_INITIAL_DIR=${DIST_DIR}/${PMD_FILE}
 PMD_DIR=${DIST_DIR}/pmd
+PMD_CHECKSUM=${DIST_DIR}/.pmd-checksum
+CHECKSUM_CALC="tar -cf - \"${PMD_DIR}\" | md5"
```

Comment:
I'm using OSX's md5 util. I don't think this would work on Linux.
Will need to rethink this approach when I convert this script into shellJs.